ddl/reorg.go

```diff
@@ -242,6 +242,15 @@ func (d *ddl) delKeysWithPrefix(prefix kv.Key, jobType JobType, job *model.Job,
 	return count, nil
 }
 
+// addFinishInfo adds schema version and table information that are used to binlog.
+func addFinishInfo(job *model.Job, ver int64, tblInfo *model.TableInfo) {
```

Comment:
I don't want this information embed model.Job. Because job's arguments are different for each operation and here only for binlog.